kernel/enter.c

```diff
@@ -655,6 +655,7 @@ static void _print_boottime(void)
 
 /* the main thread is the only thread that is not on the heap */
 static myst_thread_t _main_thread;
+myst_process_t* myst_main_process = 0;
```

Comment:
Nit: maybe this definition should go in ``thread.c`` since the declaration is in ``thread.h``.